src/app/data/services/local_storage_service.spec.ts

```diff
@@ -0,0 +1,45 @@
+///<reference path="../../../../typings/browser/ambient/jasmine/index.d.ts"/>
+///<reference path="../../mocks/ajax_service_mock_data.ts"/>
+///<reference path="local_storage_service.ts"/>
+///<reference path="../models/content_block.ts"/>
+
+
+describe('LocalStorageService', function () {
+  const service = new LocalStorageService();
+  const hash:AjaxContentBlocksHash =  { blub: <any> { blub: '1' } };
+  const hash2:AjaxContentBlocksHash = { blub: <any> { blub2: '2' } };
+
+  describe('The content block saver', function () {
+    beforeEach(function() {
+      localStorage.clear();
```

Comment:
with this we can actually put the key to the top, right?
actually, wouldn't it be better if we changed the beforeEach to afterEach, so we clean up the local storage after filling it?